apps/web/components/ui/PurchaseCard/PurchaseCard.vue

```diff
@@ -68,7 +68,7 @@
         <UiQuantitySelector
           :value="quantitySelectorValue"
           @change-quantity="changeQuantity"
-          class="min-w-[145px] flex-grow flex-shrink-0 basis-0"
+          class="min-w-[145px] flex-grow-0 flex-shrink-0 basis-0"
```

Comment:
@rbedeoan-plenty 

Can you take a look if we still need this please ?

Reply:
yes, is needed, when you click add to cart it changes width (flex-grow-0  prevents this)